apps/studio/src/features/editing-experience/components/RootStateDrawer.tsx

```diff
@@ -122,7 +154,56 @@ export default function RootStateDrawer() {
     ],
   )
 
-  const pageLayout: string = savedPageState.layout
+  const handleConversionToIndexPage = useCallback(() => {
+    const newPageState = {
+      ...savedPageState,
+      content: [],
```

Comment:
[issue] won't this essentially discard all their content? i think we shuold preserve this as we'll be causing irreversible data loss otherwise

Reply:
Hmm yeah this was intended, can see the copy used:

<img width="739" alt="image" src="https://github.com/user-attachments/assets/93fabff2-afab-4f31-bf89-692327d8e8ba" />

Reply:
Updated in 64ef2a0fa0fe05d4cdd39f8b2507df40466176ae!